users/usersRoutes.js

```diff
@@ -0,0 +1,197 @@
+const express = require('express');
+
+const usersDB = require('../data/helpers/userDb');
+
+const router = express.Router();
+
+const clickWatchLogger = require('../data/middleware');
+
+const sendError = (status, message, res) => {
+  res.status(status).json({ errorMessage: message });
+};
+
+router.post('/', clickWatchLogger, (req, res) => {
```

Comment:
I'd recommend thinking about all the endpoints you might want to build when you're assigning the routes for these requests. A post request to '/' might work fine if you're dealing with limited types of data, but what if you wanted to also add a post request for tags/posts in addition to users? How would you differentiate between those different types of data that might be the subject of a GET/POST/etc?